lib/metadata/wrapper.js

```diff
@@ -117,9 +117,9 @@ const metadata = {
         });
     },
 
-    listObject: (bucketName, prefix, marker, delimiter, maxKeys, log, cb) => {
+    listObject: (bucketName, listingParams, log, cb) => {
```

Comment:
Why this change? The previous signature was explicit and avoided creating/allocating an object.